mirage-net-macosx.opam

```diff
@@ -8,11 +8,13 @@ dev-repo:    "https://github.com/mirage/mirage-net-macosx.git"
 doc:         "https://mirage.github.io/mirage-net-macosx/"
 
 license: "ISC"
-build: ["ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%"]
+build: [
+  [ "jbuilder" "build" "--only-packages=mirage-net-macosx" ]
```

Comment:
this can also be 
```
  ["jbuilder" "build" "-p" name "-j" jobs]
```

with following constraint
```
  "jbuilder" {build & >="1.0+beta7"}
```

(for future releases)